recipes/default.rb

```diff
@@ -8,13 +8,27 @@
 include_recipe 'apt'
 include_recipe 'java'
 
-link "/etc/alternatives/java" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
-end
+#link "/etc/alternatives/java" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
+#end
+
+#link "/etc/alternatives/javac" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+#end
+
+include_recipe 'build-essential'
 
-link "/etc/alternatives/javac" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+case node['platform']
+when 'debian', 'ubuntu'
+  package 'ruby-dev' do
```

Comment:
I agree a simple script could be written to start / manage the service, but after researching Bluepill, I think it should like something that would be beneficial to use.

I wish Bluepill would manage these dependencies in its recipes, but if they have to be here, that's fine.